modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase

```diff
@@ -139,6 +139,41 @@ definition {
 			actionName = "Action Name");
 	}
 
+	@description = "LPS-145665 - Verify that you can edit the Action name"
+	@priority = "4"
+	test CanEditActionName {
+		ObjectAdmin.addObjectViaAPI(
+			labelName = "Custom Object 145",
+			objectName = "CustomObject145",
+			pluralLabelName = "Custom Objects 145");
+
+		ObjectAdmin.openObjectAdmin();
+
+		ObjectPortlet.selectCustomObject(label = "Custom Object 145");
+
+		ObjectAdmin.goToActionsTab();
+
+		ObjectAdmin.addObjectActionViaUI(
+			actionName = "Custom Action",
+			active = "true",
+			thenAction = "Webhook",
+			urlAction = "www.liferay.com",
+			whenAction = "On After Add");
+
+		ObjectAdmin.assertObjectActions(actionActive = "Custom Action");
+
+		ObjectAdmin.assertObjectActions(actionActive = "Yes");
```

Comment:
We should be asserting this when we create the action. Since we're editing the action for the test, we can assuming that it's active.